### ISSUE TITLE
Links open in Edge instead of my default browser

### ISSUE THREAD

**Issue by PhotoKevin** [apache/netbeans#6455]
### Apache NetBeans version

Apache NetBeans 19

### What happened

URLs open in Edge, not in my default browser (Vivaldi)

### How to reproduce

All URLs seem to be affected. The simplest way to reproduce is Help->Online Support

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 10 

### JDK

openjdk 19.0.2 2023-01-17 OpenJDK Runtime Environment (build 19.0.2+7-44) OpenJDK 64-Bit Server VM (build 19.0.2+7-44, mixed mode, sharing)

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

I realize this may be very difficult due to Microsoft's continuing attempts to ignore my wishes on browser usage.  I will, however, note that Libre Office Writer opens its online help in my default browser.

### Are you willing to submit a pull request?

No

**Comment by mbien** [comment]
does this open the default browser on your system? if yes then this is probably easily fixable:
```java
public class Mavenproject1 {
    public static void main(String[] args) throws URISyntaxException, IOException {
        Desktop.getDesktop().browse(new URI("https://netbeans.apache.org"));
    }
}
```
added https://github.com/apache/netbeans/labels/Contribution%20welcome + https://github.com/apache/netbeans/labels/good%20first%20issue labels

**Comment by PhotoKevin** [comment]
Yes, that opens in my default browser.

Sorry for the slow response, I was out of town and unable to try it.

**Comment by mbien** [comment]
no worries. Could you start NetBeans with `netbeans -Jorg.netbeans.modules.extbrowser.UseDesktopBrowse=true` set and check if it has the desired effect? Since it should use the same method as you tested above.

you could also add this to the `netbeans_default_options` property in `etc/netbeans.conf` or as program argument

**Comment by PhotoKevin** [comment]
That didn't work:

`C:\Program Files\NetBeans-19\netbeans\bin>netbeans -Jorg.netbeans.modules.extbrowser.UseDesktopBrowse=true

The launcher has determined that the parent process has a console and will reuse it for its own console output.
Closing the console will result in termination of the running program.
Use '--console suppress' to suppress console output.
Use '--console new' to create a separate console window.
WARNING: package com.apple.eio not in java.desktop
WARNING: package sun.awt.X11 not in java.desktop
WARNING: package com.sun.java.swing.plaf.gtk not in java.desktop
Error: Could not find or load main class org.netbeans.modules.extbrowser.UseDesktopBrowse=true
Caused by: java.lang.ClassNotFoundException: org.netbeans.modules.extbrowser.UseDesktopBrowse=true`

**Comment by mbien** [comment]
I am sorry I forgot the `-D`:
`C:\Program Files\NetBeans-19\netbeans\bin>netbeans -J-Dorg.netbeans.modules.extbrowser.UseDesktopBrowse=true`

**Comment by PhotoKevin** [comment]
The command line version worked first try.

The netbeans.conf worked too, but I got lost for a while in the difference between the one in Program Files and the one in Virtual Store under AppData. That's totally a Windows issue.

Just as a caution, I'd say that this shouldn't be closed just because there is a workaround. People doing a fresh install have an expectation that their default browser will be honored. 

Thanks,
Kevin

**Comment by mbien** [comment]
yeah. I hoped I could enable the flag by default but this would likely cause other issues as soon more is done than simple "open in browser" actions.
